exchanges/deribit/deribit_wrapper.go

```diff
@@ -0,0 +1,1132 @@
+		Exchange:        d.Name,
+		PostOnly:        orderInfo.PostOnly,
+		Price:           orderInfo.Price,
+		Amount:          orderInfo.Amount,
+		ExecutedAmount:  orderInfo.FilledAmount,
+		Fee:             orderInfo.Commission,
+		RemainingAmount: orderInfo.Amount - orderInfo.FilledAmount,
+		OrderID:         orderInfo.OrderID,
+		Pair:            pair,
+		LastUpdated:     time.UnixMilli(orderInfo.LastUpdateTimestamp),
+		Side:            orderSide,
+		Type:            orderType,
+		Status:          orderStatus,
+	}
+	return resp, nil
+}
+
+// GetDepositAddress returns a deposit address for a specified currency
+func (d *Deribit) GetDepositAddress(ctx context.Context, cryptocurrency currency.Code, accountID, chain string) (*deposit.Address, error) {
+	var addressData *DepositAddressData
+	var err error
+	if d.Websocket.CanUseAuthenticatedWebsocketForWrapper() {
+		addressData, err = d.WSRetriveCurrentDepositAddress(cryptocurrency.String())
+	} else {
+		addressData, err = d.GetCurrentDepositAddress(ctx, cryptocurrency.String())
+	}
+	if err != nil {
+		return nil, err
+	}
+	return &deposit.Address{
+		Address: addressData.Address,
+		Chain:   addressData.Currency,
+	}, nil
+}
+
+// WithdrawCryptocurrencyFunds returns a withdrawal ID when a withdrawal is
+// submitted
+func (d *Deribit) WithdrawCryptocurrencyFunds(ctx context.Context, withdrawRequest *withdraw.Request) (*withdraw.ExchangeResponse, error) {
+	err := withdrawRequest.Validate()
+	if err != nil {
+		return nil, err
+	}
+	var withdrawData *WithdrawData
+	if d.Websocket.CanUseAuthenticatedWebsocketForWrapper() {
+		withdrawData, err = d.WSSubmitWithdraw(withdrawRequest.Currency.String(), withdrawRequest.Crypto.Address, "", withdrawRequest.Amount)
+	} else {
+		withdrawData, err = d.SubmitWithdraw(ctx, withdrawRequest.Currency.String(), withdrawRequest.Crypto.Address, "", withdrawRequest.Amount)
+	}
+	return &withdraw.ExchangeResponse{
+		ID:     strconv.FormatInt(withdrawData.ID, 10),
+		Status: withdrawData.State,
+	}, err
+}
+
+// WithdrawFiatFunds returns a withdrawal ID when a withdrawal is submitted
+func (d *Deribit) WithdrawFiatFunds(ctx context.Context, request *withdraw.Request) (*withdraw.ExchangeResponse, error) {
+	return nil, common.ErrFunctionNotSupported
+}
+
+// WithdrawFiatFundsToInternationalBank returns a withdrawal ID when a withdrawal is submitted
+func (d *Deribit) WithdrawFiatFundsToInternationalBank(ctx context.Context, withdrawRequest *withdraw.Request) (*withdraw.ExchangeResponse, error) {
+	return nil, common.ErrFunctionNotSupported
+}
+
+// GetActiveOrders retrieves any orders that are active/open
+func (d *Deribit) GetActiveOrders(ctx context.Context, getOrdersRequest *order.GetOrdersRequest) (order.FilteredOrders, error) {
+	if err := getOrdersRequest.Validate(); err != nil {
+		return nil, err
+	}
+	var resp []order.Detail
+	switch getOrdersRequest.AssetType {
+	case asset.Futures, asset.Options, asset.FutureCombo, asset.OptionCombo:
```

Comment:
From [Previous Nit](https://github.com/thrasher-corp/gocryptotrader/pull/1082/files#r1021002685) `use Supported asset method and return error`